prime-dotnet-webapi/Services/EnrolleePaperSubmissionService.cs

```diff
@@ -285,15 +285,24 @@ public async Task<bool> MatchingSubmissionExistsAsync(DateTime dateOfBirth)
                         .Any(link => link.PaperEnrolleeId == e.Id));
         }
 
+        public async Task<bool> GetIsEnrolleeLinkedAsync(int enrolleeId)
```

Comment:
Sorry I have approved but it is practice so far in our app to not have the verb "get" infront and just use IsEnrolleeLinkedAsync()

Reply:
renamed